### ISSUE TITLE
UI does not show docs inline on a full dynamic index.

### ISSUE THREAD

**Issue by fulmicoton** [quickwit-oss/quickwit#1490]
To reproduce create an index with a dynamic index and no field.

The unfolded UI works, but the inline docs are blank.

**Comment by fulmicoton** [comment]
![screenshotlog](https://user-images.githubusercontent.com/1021506/169181857-3f1c1dc7-f834-4aef-872a-00e61c8ae93a.png)

**Comment by fmassot** [comment]
can you share your index config?

**Comment by fulmicoton** [comment]
Full dynamic

```yaml
version: 0
index_id: wikijson
doc_mapping:
  mode: dynamic
```

**Comment by fmassot** [comment]
Ok I have the same config. I indexed 1000 Wikipedia articles and the backend returns documents like this one:
```json
{
  "body": "\nJeon Hye-jin (actress, born 1988)\n\nJeon Hye-jin (born June 17, 1988) is a South Korean actress.\nPersonal life.\nJeon married his \"Smile, You\" co-star Lee Chun-hee on March 11, 2011. Their daughter, Lee So Yu, was born on July 30, 2011.\n\n",
  "title": "Jeon Hye-jin (actress, born 1988)",
  "url": "https://en.wikipedia.org/wiki?curid=48687903"
}
```

I guess your documents are a bit different. I ingested also documents like yours but I can't reproduce it.

Maybe you run the UI with an old build, normally I fixed this issue a few days ago.

**Comment by fulmicoton** [comment]
ok let me try again.

**Comment by fulmicoton** [comment]
I confirm this is already fixed.
